crypto/x509/x509_req.c

```diff
@@ -115,6 +115,13 @@ EVP_PKEY *X509_REQ_get_pubkey(X509_REQ *req)
     return (X509_PUBKEY_get(req->req_info.pubkey));
 }
 
+EVP_PKEY *X509_REQ_get0_pubkey(X509_REQ *req)
+{
+    if (req == NULL)
+        return (NULL);
```

Comment:
remove the parens.  i prefer using ```?:``` but I think most don't so that part is okay.